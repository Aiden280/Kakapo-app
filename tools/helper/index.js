import fs from 'fs-extra';

// Upload stats.json to `http://webpack.github.io/analyse` for analysis.
// Also you can analyze size with `https://github.com/robertknight/webpack-bundle-size-analyzer`:
// `cat stats.json | analyze-bundle-size`
export const writeStats = stats =>
  fs.writeFile('stats.json', JSON.stringify(stats.toJson(), null, 4));

export default {};
