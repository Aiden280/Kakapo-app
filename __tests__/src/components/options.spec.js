import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import { ImportOptions } from 'components/';
import { getData } from '../helper';

test('<Options/> render', () => {
  const props = {
    ...getData('intl')
  };
  const tree = renderer
    .create(
      <MemoryRouter>
        <ImportOptions {...props} />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
