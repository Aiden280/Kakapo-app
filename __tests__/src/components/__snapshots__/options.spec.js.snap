// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Options/> render 1`] = `
<div
  className="downloads"
>
  <div
    className="media"
  >
    <h4
      className="add-sounds-title"
    >
      Add a sound
    </h4>
    <ul
      className="option-ul"
    >
      <li
        className="option options-kakapo"
      >
        <a
          href="/kakapo"
          onClick={[Function]}
        >
          <i
            className="icon-img-kakapo"
          />
          <span
            className="text"
          >
            Kakapo favourites
          </span>
        </a>
      </li>
      <li
        className="option options-youtube"
      >
        <a
          href="/youtube"
          onClick={[Function]}
        >
          <i
            className="icon-img-youtube"
          />
          <span
            className="text"
          >
            YouTube
          </span>
        </a>
      </li>
      <li
        className="option options-soundcloud"
      >
        <a
          href="/soundcloud"
          onClick={[Function]}
        >
          <i
            className="icon-img-soundcloud"
          />
          <span
            className="text"
          >
            SoundCloud
          </span>
        </a>
      </li>
      <li
        className="option options-custom"
      >
        <a
          href="/custom"
          onClick={[Function]}
        >
          <i
            className="icon-img-custom icon-add"
          />
          <span
            className="text"
          >
            Custom link
          </span>
        </a>
      </li>
    </ul>
    <div
      className="divider"
    />
    <ul
      className="option-ul"
    >
      <li
        className="option"
      >
        <a
          href="/playlist"
          onClick={[Function]}
        >
          <i
            className="icon-playlist"
          />
          <span
            className="text"
          >
            Playlists
          </span>
        </a>
      </li>
      <li
        className="option"
      >
        <a
          href="/settings"
          onClick={[Function]}
        >
          <i
            className="icon-settings"
          />
          <span
            className="text"
          >
            Settings
          </span>
        </a>
      </li>
    </ul>
  </div>
</div>
`;
